linode/manager
# upcoming: [M3-7296] - Create Load Balancer Summary page

## Description 📝
Add's Load Balancer Summary page.

> [!note]
>  This PR does not support rules and pricing. I will cover that in a separate PR to keep the size down.

## Changes  🔄

![image](https://github.com/linode/manager/assets/119517080/0c2b1d66-c95a-4ac5-9e27-4b3c7ac14585)


## How to test 🧪

### Prerequisites
(How to setup test environment)
- Turn on mocks or test with the AGLB test account.
- Turn on the AGLB Full Create Flow feature flag.
- Go to http://localhost:3000/loadbalancers/create


### Verification steps 
(How to verify changes)
- Verify you can add, update, and remove LoadBalancer Label, Service Targets and Routes on the summary page of  AGLB create flow. 


## As an Author I have considered 🤔

*Check all that apply*
